profiles/nawed2611.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/83456083/178050998-5d11d3df-e779-474e-8ea9-12e8fd782968.png",
+  "issueId": 1224,
+  "name": "Nawed Ali",
+  "username": "nawed2611"
+}

```

Comment:
```suggestion
{
  "image": "https://user-images.githubusercontent.com/83456083/178050998-5d11d3df-e779-474e-8ea9-12e8fd782968.png",
  "issueId": 1224,
  "name": "Nawed Ali",
  "username": "nawed2611"
}

```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix